### PR TITLE
created freedompraise.json

### PR DESCRIPTION
**Pull request by freedompraise**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4339"></a>

### CODE REVIEW

File: data/freedompraise.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/praise-dike-7a25011b3/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/freedom_praise",
+      "icon": "FaTwitter"
+    },
```
Comment (senali-d) [inline data/freedompraise.json:22]: Remove `,`
```suggestion
    }
```

**Review by senali-d — CHANGES_REQUESTED** [review]
@freedompraise Please revert the changes in the files other than the `data/freedompraise.json` file.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts [here](https://github.com/EddieHubCommunity/LinkFree/pull/4339/conflicts). Let us know if you need any help with this. :)

**Comment by kumarsonsoff3** [comment]
Hey @freedompraise, do you need any help in making the requested changes? Feel free to let us know. :)

You can use the command line to resolve merge conflicts following these steps. 
- Step 1: From your project repository, check out a new branch and test the changes.

```
git checkout -b freedompraise-main main
git pull https://github.com/freedompraise/LinkFree.git main
```

- Step 2: Merge the changes and update on GitHub.
```
git checkout main
git merge --no-ff freedompraise-main
git push origin main
```

**Comment by eddiejaoude** [comment]
It looks like GitHub Actions are running in your fork, please disable these to avoid these generated files.

Also it is better to make changes in a branch, so I will close pull request, you can follow our quickstart guide here and create a new pull request https://linkfree.eddiehub.io/docs/quickstart

**Pull request closed without merging**